examples/with-supabase/app/login/page.tsx

```diff
@@ -45,7 +45,11 @@ export default function Login({
     });
 
     if (error) {
-      return redirect("/login?message=Could not authenticate user");
+      const { code } = error
+      if (code) {
```

Comment:
Thank you! Could you please run Prettier in the repo for these changes?

Reply:
@leerob apologies I understand @josueJURE has had two weeks of busy interviews and may not have been able to attend to this.  I'll be in touch with him to see if there is anything I can help.  I am not sure whether I can take over and submit the PR on my @3willows account.